int16 car_id
int16 lane_id
float64 t
float64 x
float64 y
float32 heading
float32 angular_V
float64 vel
float64 desired_vel
float64 acc
int16 priority
float32 length
float32 width
float32 max_V
float32 max_A
float32 min_A
float32 max_lateral_g
---
bool success
float64[] x
float64[] y
float64[] heading
float64[] vel
float64[] t